pkgs/development/libraries/maplibre-gl-native/default.nix

```diff
@@ -30,6 +31,11 @@ mkDerivation rec {
       url = "https://git.alpinelinux.org/aports/plain/testing/mapbox-gl-native/0002-skip-license-check.patch?id=6751a93dca26b0b3ceec9eb151272253a2fe497e";
       sha256 = "1yybwzxbvn0lqb1br1fyg7763p2h117s6mkmywkl4l7qg9daa7ba";
     })
+    (fetchpatch2 {
+      name = "cstdint.patch";
+      url = "https://git.alpinelinux.org/aports/plain/community/maplibre-gl-native/cstdint.patch?id=ae8edc6b02df388ef37a69c12a5df25dd8550238";
```

Comment:
Could we get the patch from upstream instead of alpine linux?
https://github.com/maplibre/maplibre-native/commit/87025882ec733832d4314554c1d01efca9dbce0b

Reply:
Or does that not apply cleanly because it wants to update the submodule instead of just modifying its contents?

Reply:
Doesn't apply.